### PR TITLE
[#127807225] Add `variations_agreed` to framework users csv download

### PR DESCRIPTION
**Pull request by TheDoubleK**
Completes this story: https://www.pivotaltracker.com/story/show/127807225

Depends on API change here: 
 - [ ] https://github.com/alphagov/digitalmarketplace-api/pull/436

### CODE REVIEW

File: tests/app/main/views/test_users.py [hunk 1]
```diff
@@ -245,6 +245,8 @@ def _assert_things_about_user_export(self, response, users):
             assert sorted(list(users[0].keys())) == sorted(rows[0])
 
             for index, user in enumerate(users):
+                print("LHS: {}".format(sorted([six.text_type(val) for val in user.values()])))
```
Comment (Wynndow) [inline tests/app/main/views/test_users.py]: Do these prints serve any purpose? They don't assert anything and they don't print out in my tests if run locally?

Comment (TheDoubleK) [inline tests/app/main/views/test_users.py]: No that was some debug stuff I forgot to remove - they do print out if the test fails, I'll drop them now.

Comment (TheDoubleK) [inline tests/app/main/views/test_users.py]: Needed a rebase anyway so I squashed this into previous commit.

**Comment by Wynndow** [comment]
Looks good to me! 👍

**Pull request merged**